cypress/support/pageObjects/inventoryPage.js

```diff
@@ -0,0 +1,88 @@
+var inventoryItem = require('../../models/inventoryItem.json');
+
+export class InventoryPage {
+    static wrapItems() {
+        let pageItems = [];
+        let amountOfItems = this.getAmountOfItems();
+        console.log('amountOfItems', amountOfItems);
+        for (let i = 1; i <= amountOfItems; i++) {
+            inventoryItem = {name: '', price: '', description: '', addToCartLocator: ''};
+            let itemJson = {...inventoryItem};
+            cy.get(`.inventory_list > :nth-child(${i})> .inventory_item_description>.inventory_item_label > a >.inventory_item_name`)
+                .invoke('text')
+                .then((label) => {
+                    cy.wrap(label).as('label');
+                    itemJson['name'] = label;
+                    let addToCartLocator = label;
+                    addToCartLocator = addToCartLocator.toLowerCase();
+                    let amountOfSpaceSymbols = addToCartLocator.split(' ').length - 1;
+                    while (amountOfSpaceSymbols) {
+                        addToCartLocator = addToCartLocator.replace(' ', '-');
+                        amountOfSpaceSymbols -= 1;
+                    }
+                    itemJson['addToCartLocator'] = 'add-to-cart-' + addToCartLocator;
+                });
+            cy.get(`.inventory_list > :nth-child(${i})> .inventory_item_description > .pricebar > .inventory_item_price`)
+                .invoke('text')
+                .then((price) => {
+                    cy.wrap(price).as('price');
+                    itemJson['price'] = price.slice(1);
+                });
+            cy.get(`.inventory_list > :nth-child(${i})> .inventory_item_description>.inventory_item_label > .inventory_item_desc`)
```

Comment:
Extract the method "getItemDescription"

Reply:
Currently it is used as a part of switch statement
I don't think moving it to another method will improve the code